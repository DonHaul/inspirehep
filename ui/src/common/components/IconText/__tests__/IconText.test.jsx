import { render } from '@testing-library/react';
import { InfoOutlined } from '@ant-design/icons';

import IconText from '../IconText';

describe('IconText', () => {
  it('renders with all props set', () => {
    const { asFragment } = render(
      <IconText icon={<InfoOutlined />} text="Test" className="test" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
