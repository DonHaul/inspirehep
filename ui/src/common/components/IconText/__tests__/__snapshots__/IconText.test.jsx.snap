// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconText renders with all props set 1`] = `
<DocumentFragment>
  <span
    class="__IconText__ test"
  >
    <span
      class="icon"
    >
      <span
        aria-label="info"
        class="anticon anticon-info"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="info"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M448 224a64 64 0 10128 0 64 64 0 10-128 0zm96 168h-64c-4.4 0-8 3.6-8 8v464c0 4.4 3.6 8 8 8h64c4.4 0 8-3.6 8-8V400c0-4.4-3.6-8-8-8z"
          />
        </svg>
      </span>
    </span>
    <span
      class="v-top"
    >
      Test
    </span>
  </span>
</DocumentFragment>
`;
